src/goto-programs/slice_function_calls.cpp

```diff
@@ -105,4 +126,50 @@ void slice_function_callst::operator()(goto_model_functiont &goto_function)
 #endif
 }
 
+slice_function_callst::variable_bounds_mapt
+slice_function_callst::compute_variable_bounds(
```

Comment:
Just noting that  `goto_program2codet::dead_map` does something similar. It would be good to converge on one, shared implementation.